scripts/ai-help.sql

```diff
@@ -31,6 +31,7 @@ create table
     title text not null,
     mdn_url text not null,
     html text null,
+    markdown text null,
```

Comment:
This should go to the end, order matters here, doesn't it?

Reply:
This file is mainly informational, so I think it doesn't have to mirror the column order in the database (since Postgres doesn't support adding columns at a specific position).